observatorium/observatorium.yaml

```diff
@@ -0,0 +1,151 @@
+---
+apiVersion: core.observatorium.io/v1alpha1
+kind: Observatorium
+metadata:
+  labels:
+    app.kubernetes.io/instance: opf-observatorium
+    app.kubernetes.io/name: observatorium-cr
+    app.kubernetes.io/part-of: observatorium
+  name: opf-observatorium
+spec:
+  # Observatorium API is scaled down
+  api:
+    image: quay.io/observatorium/observatorium:master-2020-09-08-v0.1.1-142-g61a908f
+    rbac:
+      roleBindings:
+      - name: test
+        roles:
+        - read-write
+        subjects:
+        - kind: user
+          name: admin@example.com
+      roles:
+      - name: read-write
+        permissions:
+        - read
+        - write
+        resources:
+        - logs
+        - metrics
+        tenants:
+        - test
+    replicas: 0
+    tenants:
+    - id: 1610b0c3-c509-4592-a256-a1871353dbfa
+      name: test
+      oidc:
+        clientID: test
+        clientSecret: ZXhhbXBsZS1hcHAtc2VjcmV0
+        issuerURL: http://dex.dex.svc.cluster.local:5556/dex
+        usernameClaim: email
+    tls:
+      caKey: ca.pem
+      certKey: cert.pem
+      configMapName: observatorium-opf-tls
+      keyKey: key.pem
+      secretName: observatorium-opf-tls
+      serverName: observatorium-opf-observatorium-api.opf-monitoring.svc.cluster.local
+    version: master-2020-09-08-v0.1.1-142-g61a908f
+  apiQuery:
+    replicas: 0
+    image: quay.io/thanos/thanos:master-2020-08-12-70f89d83
+    version: master-2020-08-12-70f89d83
+  compact:
+    enableDownsampling: false
+    image: quay.io/thanos/thanos:master-2020-08-12-70f89d83
+    replicas: 0
+    retentionResolution1h: 1s
+    retentionResolution5m: 1s
+    retentionResolutionRaw: 14d
+    version: master-2020-08-12-70f89d83
+    volumeClaimTemplate:
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+  hashrings:
+  - hashring: default
+    tenants: []
+  # Only Loki Components are scaled up
+  loki:
+    image: quay.io/4n4nd/loki:1.6.1
+    replicas:
+      distributor: 1
+      ingester: 1
+      querier: 1
+      query_frontend: 1
+      table_manager: 1
+    version: 1.6.1
+    volumeClaimTemplate:
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 250Mi
+  objectStorageConfig:
+    loki:
+      accessKeyIdKey: aws_access_key_id
+      bucketsKey: buckets
+      endpointKey: endpoint
+      regionKey: region
+      secretAccessKeyKey: aws_secret_access_key
+      secretName: loki-objectstorage
```

Comment:
This secret is missing from the manifests completely.

Reply:
This is a secret that has been deployed manually